docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -875,6 +875,19 @@ When the flag [remote_bootstrap_from_leader_only](#remote-bootstrap-from-leader-
 
 Default: `5`
 
+##### --db_block_cache_num_shard_bits
```

Comment:
should it not be under sharding flags? - https://deploy-preview-20556--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-tserver/#sharding-flags

Reply:
No, this creates the block cache inside 1 server to work with more cpus on the server. It's not cluster sharding.